service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsAboutToSubmitControllerTest.java

```diff
@@ -91,6 +97,7 @@ void before() {
         given(requestData.userRoles()).willReturn(Set.of(ADMIN_ROLE));
         given(idamClient.getUserDetails(eq(USER_AUTH_TOKEN))).willReturn(createUserDetailsWithHmctsRole());
         given(documentConversionService.convertToPdf(UPLOADED_DOCUMENT)).willReturn(PDF_DOCUMENT);
+        given(userService.isHmctsUser()).willReturn(true);
```

Comment:
Looks like LA solicitor can upload additional applications as well. Will there any test case for LA solicitor journey?

Reply:
all of those confidential handling are processed in the service. Unit test of services have covered those LA and solicitor scenarios. Does that make sense?